napari/plugins/npe2api.py

```diff
@@ -44,7 +47,7 @@ def _user_agent() -> str:
 
 
 class SummaryDict(TypedDict):
-    """Objects returned at https://npe2api.vercel.app/api/summary ."""
+    """Objects returned at https://npe2api-5lai6hg8a-napari.vercel.app/api/extended_summary ."""
```

Comment:
Uhhh, where is this terrible looking string coming from? Is this a versioning thing? I also notice on line 64 we're pinging `"https://npe2api.vercel.app/api/extended_summary"` so I'm curious about this

Reply:
This was included before https://github.com/napari/npe2api/pull/11 got merged.  It can be simplified to the `"https://npe2api.vercel.app/api/extended_summary"` now.